SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -137,13 +137,13 @@ UnitTest {
 		^boolean
 	}
 
-	assertEquals { |a, b, message = "", report = true, onFailure |
-		this.assert( a == b, message + "\nIs:\n\t" + a + "\nShould be:\n\t" + b + "\n", report, onFailure)
+	assertEquals { |got, expected, message = "", report = true, onFailure |
```

Comment:
This is an API change.

Reply:
Yeah. I highly doubt there is anyone out there is using these, but it's probably better to just leave it as is.

Reply:
Forgive my ignorance - presumably this is because parameters can be named in calls?

I can easily drop this commit - the others are hopefully non-contentious.

Reply:
A related question is - how / when can we make API changes?  Not until 4.0?

Reply:
Yeah, it's because parameter names can be used in client code. Sorry, should have explained that.

I think the general consensus is that we avoid changing the public API unless it's reached a high pain point. There's no scheduled plan for releasing 4.0, so it would be disingenuous of me to say "not until 4.0". We're ok with some breaking changes between minor releases I think. Sorry if that doesn't do a great job of answering your question, but as a community we haven't made a decision about that yet.

Reply:
>We're ok with some breaking changes between minor releases I think. 

yes, for 3.10 definitely these parameter names should be changed (I also keep wondering which is which).

Reply:
OK great, I can definitely pull this commit out into a separate PR which can be deferred to 3.10 if that's the preferred route.

Reply:
> I can definitely pull this commit out into a separate PR which can be deferred to 3.10 if that's the preferred route.

Sure, that would be great! Thanks.

Reply:
Done.